deployment/environments/terraform-development.tfvars

```diff
@@ -0,0 +1,58 @@
+project = "OpenSupplyHub"
+environment = "Development"
+py_environment = "Staging"
+
+aws_region = "eu-west-1"
+aws_availability_zones = ["eu-west-1a", "eu-west-1b"]
+
+r53_private_hosted_zone = "osh.internal"
+r53_service_discovery_zone = "sd.internal"
+r53_public_hosted_zone = "os-hub.net"
+
+cloudfront_price_class = "PriceClass_All"
+
+bastion_ami = "ami-0bb3fad3c0286ebd5"
+bastion_instance_type = "t3.nano"
+
+rds_allocated_storage = "128"
+rds_engine_version = "12"
+rds_parameter_group_family = "postgres12"
+rds_instance_type = "db.t3.micro"
+rds_database_identifier = "opensupplyhub-enc-stg"
+rds_database_name = "opensupplyhub"
```

Comment:
could that be looked up using **data** or **module.database_enc** or templated by variables like **${var.project}-${var.environments}-db** in TF code?

Reply:
Probably yes, but this is definitely different task

Reply:
Also what for? what's advantage of using different database names for different environments?

Reply:
Just for the recognition while you parse some logs. If you don't want this - you can skip "var.env" part

Reply:
https://opensupplyhub.atlassian.net/browse/OSDEV-937